addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/station/NAMainHandler.java

```diff
@@ -98,11 +98,9 @@ protected State getNAThingProperty(String channelId) {
                             dashboardData.getHumidity());
                     return toDecimalType(WeatherUtils.getDewPointDep(dashboardData.getTemperature(), dewPoint));
                 case CHANNEL_WIND_UNIT:
-                    return userAdministrative != null ? new DecimalType(userAdministrative.getWindunit())
```

Comment:
Be aware: The `null` check here was against `userAdministrative`. The new wrapper method checks against the return value of the `userAdministrative.getWindunit()`. Do we have to check both to prevent  another possible NPE?

Reply:
I checked the original issue, the NPE was caused by a `null` `userAdministrative.getWindunit()` and not by a `null` `userAdministrative`, that is why I changed it back.

Reply:
Yes, I know. Let's hope we always have access to a valid `userAdministrative` object.